.circleci/config.yml

```diff
@@ -117,6 +129,22 @@ workflows:
           requires:
             - build
 
+  build-test-publish:
```

Comment:
**Question**: Why is this new CI stage needed?

Reply:
this workflow will be triggered when a new tag is published, and this will run the jobs required to publish on npm